cli/cmd/root.go

```diff
@@ -23,6 +23,11 @@ Further info is available at https://github.com/up9inc/mizu`,
 		if err := config.InitConfig(cmd); err != nil {
 			logger.Log.Fatal(err)
 		}
+
+		if err := config.Config.Validate(); err != nil {
```

Comment:
why not validate when Init?

Reply:
because you have to do it after the config is initiated